app/Http/Routes/Auth.php

```diff
@@ -0,0 +1,5 @@
+<?php
+Route::auth();
+Route::get('password/email', 'Auth\PasswordController@getEmail');
```

Comment:
the route for `password` is already setup in `Route::auth()`. I think you don't need to create again. If you want to custom something, just override in `AuthController` class